packages/examples/html/src/index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
```

Comment:
@DeMoorJasper should we just combine this with the hmr example now that the hmr example serves an html file?